httpmw/logging/logging.go

```diff
@@ -0,0 +1,71 @@
+package logging
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+type Logger interface {
```

Comment:
Do it in another PR.

Either move this to a different package that can be used by the gPRC middleware
Or just use the slogger interface if it is already there. Arbitrarily defining two functions doesn't seem a well thought interface.

Reply:
Added a todo to add in another PR. Will, implement something similar to the article below.

https://medium.com/@ansujain/building-a-logger-wrapper-in-go-with-support-for-multiple-logging-libraries-48092b826bee